_docs/install/openshift/preprovisioned.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: guide
+title: StorageOS Docs - OpenShift Pre-provisioned Volumes
+anchor: install
+module: install/openshift/preprovisioned
+---
+
+# Pre-provisioned Persistent Volumes
+
+1. Create a volume using the StorageOS CLI or API.  Consult the
+   [volume documentation]({% link _docs/manage/volumes/index.md %}) for details.
+
+   ```bash
+   # Create a volume with one replica.
+   storageos volume create nginx-pv01 --label storageos.com/replicas=1
+
+   default/nginx-pv01
+   ```
+
+1. Create the persistent volume `nginx-vol01`.
+
+   ```bash
+   cat > storageos-pv.yaml <<EOF
+   ---
+   apiVersion: v1
+   kind: PersistentVolume
+   metadata:
+     name: pv0001
+   spec:
+     capacity:
+       storage: 5Gi
+     accessModes:
+       - ReadWriteOnce
+     persistentVolumeReclaimPolicy: Recycle
+     storageos:
+       # This volume must already exist within StorageOS
+       volumeName: nginx-pv01
+       # volumeNamespace is optional, and specifies the volume scope within
+       # StorageOS.  Set to `default` or leave blank if you are not using
+       # namespaces.
+       volumeNamespace: default
+       # The filesystem type to create on the volume, if required.
+       fsType: ext4
+   ...
```

Comment:
...?  Is that a yaml thing?

Reply:
Not sure - copied from the existing docs. @senax?